p/themes/Ansum/ansum.css

```
@@ -1308,6 +1311,4 @@ body.register {
 a, button.as-link {
 	outline: none;
 	color: #ca7227;
-}
-
-/*# sourceMappingURL=ansum.css.map */
+}
```

Please keep the newline, not really sure what the point of the commented map thing was. :-)
```suggestion
}

```

---

thanks, I have overseen it